manifests/m/mar10/WsgiDAV/3.1.1.0/mar10.WsgiDAV.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: mar10.WsgiDAV
+PackageVersion: 3.1.1.0
+PackageLocale: en-US
+Publisher: Martin Wendt

```

Comment:
```suggestion
Publisher: Martin Wendt, Ho Chun Wei
```

Reply:
Windows Package Manager requires the `Publisher:` to match Control Panel for mapping the application to `winget list` and allow other people to upgrade via the `winget upgrade` command:
![image](https://user-images.githubusercontent.com/11600822/134743206-d99ed8a1-3e81-4013-ba20-89e1d0017242.png)
![image](https://user-images.githubusercontent.com/11600822/134743216-f738d82c-38b0-488a-bb67-61e13376a3aa.png)